admin/server/billing.go

```diff
@@ -233,6 +237,17 @@ func (s *Server) CancelBillingSubscription(ctx context.Context, req *adminv1.Can
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// err = s.admin.Email.SendSubscriptionCancelled(&email.SubscriptionCancelled{
```

Comment:
can you uncomment this, just remove the `Plan Name' from template no harm in that, once Eric O is back we can discuss.